Server/mods/deathmatch/logic/luadefs/CLuaWorldDefs.cpp

```diff
@@ -983,6 +999,11 @@ int CLuaWorldDefs::setFogDistance(lua_State* luaVM)
     return 1;
 }
 
+bool CLuaWorldDefs::SetGrassDrawDistance(float closeDistance, float farDistance)
+{
+    return CStaticFunctionDefinitions::SetGrassDrawDistance(closeDistance, farDistance);
```

Comment:
If the function always returns true, then there’s no reason for it to be a bool - it should be void and return nil.